### PR TITLE
Adapter creation

### PR DESCRIPTION
**Pull request by Heavy32**

### CODE REVIEW

File: Adapter/StorageAdapter.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        private readonly Storage storage;
+
+        public StorageAdapter()
+        {
+            storage = new Storage();
+        }
+
+        public void GetFormattedItem()
+        {
+            Console.WriteLine("--------------");
```
Comment (alexzaitzev) [inline Adapter/StorageAdapter.cs]: It's not decorator, no need to decorate the output

File: Adapter/StorageAdapter.cs [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+namespace Adapter
+{
+    class StorageAdapter : IStorage
+    {
+        private readonly Storage storage;
+
+        public StorageAdapter()
+        {
+            storage = new Storage();
```
Comment (alexzaitzev) [inline Adapter/StorageAdapter.cs]: Please use contructor injector where applicable

File: Decorator/CarDecorator/BaseCar.cs [hunk 3]
```diff
@@ -0,0 +1,18 @@
+﻿using System;
```
Comment (alexzaitzev) [inline Decorator/CarDecorator/BaseCar.cs]: Delete all others files form this PR, fork from master all branches

**Review by alexzaitzev — CHANGES_REQUESTED** [review]
See the comments

**Review by alexzaitzev — APPROVED** [review]

File: Adapter/StorageAdapter.cs (hunk 1 above)
Comment (Heavy32) [reply Adapter/StorageAdapter.cs]: done

File: Adapter/StorageAdapter.cs (hunk 2 above)
Comment (Heavy32) [reply Adapter/StorageAdapter.cs]: done

File: Decorator/CarDecorator/BaseCar.cs (hunk 3 above)
Comment (Heavy32) [reply Decorator/CarDecorator/BaseCar.cs]: done

File: Adapter/Program.cs [hunk 4]
```diff
@@ -1,34 +1 @@
-﻿using System;
-
-namespace Adapter
-{   //NOT FINISHED YET
```
Comment (alexzaitzev) [inline Adapter/Program.cs:4]: Your PR is ready or it's not finished yet?

**Review by alexzaitzev — APPROVED** [review]

**Pull request merged**